TrackletAlgorithm/TrackMergerTop.cc

```diff
@@ -0,0 +1,6 @@
+#include "TrackMergerTop.h"
+
+void TrackMergerTop(const BXType bx, TrackFit::Trackword trackWord [kMaxProc], TrackFit::BarrelStubWord barrelStubWords[4][kMaxProc], 
+TrackFit::DiskStubWord diskStubWords[4][kMaxProc])){
+    TrackMerger(bx, inputTracks, outputTracks);
```

Comment:
`inputTracks` and `outputTracks` are undefined here.

I think you just want to pass the arguments of `TrackMergerTop` directly to `TrackMerger`. It seems kind of silly right now, but eventually `TrackMerger` will probably gain some template parameters, and we will need this wrapper function because HLS doesn't allow top functions that have template parameters.